Tests iOS/Support/Response+factories.swift

```diff
@@ -31,8 +31,11 @@ extension Response {
         fixture(named: "slate-detail-\(number)")
     }
 
-    static func saveItem(_ fixtureName: String = "save-item") -> Response {
-        fixture(named: fixtureName)
+    static func saveItem(_ fixtureName: String = "save-item", number: Int? = nil) -> Response {
```

Comment:
I see what you're going for here but I think this API could get a bit confusing. If I pass a number, it's going to add it to a default fixture name, but if I pass a number and a fixture name then it will append the number to the fixture name I just passed. In the latter case, I would probably opt to pass the whole fixture name, e.g.: `saveItem("save-item-2")`

What do you think about taking out the `number: Int` param and just passing in "save-item-2" as the fixture name in your tests?

Reply:
makes sense! will update